xbmc/utils/test/TestSysfsUtils.cpp

```diff
@@ -0,0 +1,50 @@
+/*
+ *  Copyright (C) 2005-2018 Team Kodi
+ *  This file is part of Kodi - https://kodi.tv
+ *
+ *  SPDX-License-Identifier: GPL-2.0-or-later
+ *  See LICENSES/README.md for more information.
+ */
+
+#include "utils/SysfsUtils.h"
+
+#include "gtest/gtest.h"
```

Comment:
wrong include format, gtest is an external library -> `#include <gtest/gtest.h>`